### PR TITLE
[Chore](compile) add some compile_check_begin

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
add some compile_check_begin

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/pipeline/local_exchange/local_exchanger.cpp [hunk 1]
```diff
@@ -170,11 +170,11 @@ Status ShuffleExchanger::get_block(RuntimeState* state, vectorized::Block* block
 Status ShuffleExchanger::_split_rows(RuntimeState* state, const uint32_t* __restrict channel_ids,
                                      vectorized::Block* block,
                                      LocalExchangeSinkLocalState& local_state) {
-    const auto rows = block->rows();
+    const auto rows = int32_t(block->rows());
```
Comment (Gabriel39) [inline be/src/pipeline/local_exchange/local_exchanger.cpp:173]:
```suggestion
    const auto rows =cast_set<int32_t>(block->rows());
```

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Review by Mryange — APPROVED** [review]

**Pull request merged**